utils/CssParser.js

```diff
@@ -0,0 +1,99 @@
+const FileHelper = require('../helpers/FileHelper');
+const stringInject = require('stringinject');
```

Comment:
Add this package to the package.json

Reply:
Hi, I removed this one since it is not used anymore